R/glmabc.R

```diff
@@ -42,7 +42,7 @@
 glmabc = function(formula, family = stats::gaussian, data, ..., props = NULL){
 
 	# Fill in the data argument with stats::model.frame
-	data <- model_frame(formula = formula, data = data)
+	data <- model_frame(formula = formula, data = data, family = family, props = props)
```

Comment:
Will this run into problems if the "..." argument in glmabc has more terms that are passed in?